experiments/cell/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { CodeCellModel, CodeCell } from '@jupyterlab/cells';
+import { RenderMimeRegistry, standardRendererFactories } from '@jupyterlab/rendermime';
+
+import '@jupyterlab/cells/style/index.css';
+import '@jupyterlab/theme-light-extension/style/index.css';
+
+import '../style/index.css';
+
+const NUMBER_OF_CELLS = 1000;
+
+const editorContent = document.querySelector("#editor-content");
+const cells = new Array<CodeCell>();
+
+for (let i = 0; i < NUMBER_OF_CELLS; i++) {
+  const cell = new CodeCell({
+    rendermime: new RenderMimeRegistry({
+      initialFactories: standardRendererFactories
+    }),
+    model: new CodeCellModel({})
+  }).initializeState();
+  cells.push(cell);
+}
+
+cells.forEach((cell, index) => {
+  cell.activate();
+  console.log(`activate ${index}`, performance.now());  
```

Comment:
where does the `performance.now()` come from 🙃 ?

Reply:
This is standard api to have fine-grained measures

Reply:
👍